core/src/main/scala/kafka/admin/ConfigCommand.scala

```diff
@@ -41,7 +41,7 @@ object ConfigCommand {
     val opts = new ConfigCommandOptions(args)
 
     if(args.length == 0)
-      CommandLineUtils.printUsageAndDie(opts.parser, "Add/Remove entity (topics/clients) configs")
+      CommandLineUtils.printUsageAndDie(opts.parser, "Add/Remove entity (topics/clients/broker) configs")
```

Comment:
Nitpick: should be `brokers` to be consistent with the other ones? Or should the other ones be made singular?